extensions/http/src/main/java/org/mule/extension/http/api/listener/HttpListenerProvider.java

```diff
@@ -106,6 +118,10 @@
     @Inject
     private MuleContext muleContext;
 
+    private ThreadingProfile workerThreadingProfile;
```

Comment:
It should in a way for compatibility, but since the whole threading thing is going to be reviewed I decided to avoid it for now.